eclair-core/src/main/scala/fr/acinq/eclair/wire/internal/channel/version0/ChannelCodecs0.scala

```diff
@@ -335,7 +335,7 @@ private[channel] object ChannelCodecs0 {
 
     val fundingCreatedCodec: Codec[FundingCreated] = (
       ("temporaryChannelId" | bytes32) ::
-        ("fundingTxid" | bytes32) ::
+        ("fundingTxHash" | txIdAsHash) ::
```

Comment:
At the codec level, I think we should describe what is encoded, shouldn't we? That's why I made this a `fundingTxHash` because we do encode it as a hash?

Reply:
Fair enough!